WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadViewModel.kt

```diff
@@ -1,15 +1,130 @@
 package org.wordpress.android.ui.jetpack.backup
 
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
 import androidx.lifecycle.ViewModel
+import kotlinx.android.parcel.Parcelize
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadStep.COMPLETE
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadStep.DETAILS
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadStep.PROGRESS
+import org.wordpress.android.util.wizard.WizardManager
+import org.wordpress.android.util.wizard.WizardNavigationTarget
+import org.wordpress.android.util.wizard.WizardState
+import org.wordpress.android.viewmodel.SingleEventObservable
+import org.wordpress.android.viewmodel.SingleLiveEvent
 import javax.inject.Inject
 
-class BackupDownloadViewModel @Inject constructor() : ViewModel() {
-    private var isStarted: Boolean = false
-    fun start() {
-        if (isStarted) {
-            return
-        }
+const val KEY_BACKUP_DOWNLOAD_ACTIVITY_ID_KEY: String = "key_backup_download_activity_id_key"
```

Comment:
Minor (🔍 ): Consider removing explicit type specification for this constant to make it consistent with the rest.

Reply:
in ef12c38